model-optimizer/extensions/front/tf/space_to_batch.py

```diff
@@ -96,3 +97,5 @@ def find_and_replace_pattern(self, graph: Graph):
                 const_begin.out_port(0).connect(input_to_node.in_port(1))
                 diff.out_port(0).connect(input_to_node.in_port(2))
                 input_to_node.out_port(0).connect(node.in_port(idx + 1))
+                convert = Cast(graph, {'dst_type': np.int64}).create_node()
```

Comment:
Please, set the 'name' attribute for Node. Let's avoid random node names wherever possible.